samples/e2eTestUtils/src/Constants.ts

```diff
@@ -6,6 +6,8 @@ export const ENV_VARIABLES = {
 
 export const LAB_API_ENDPOINT = "https://msidlab.com/api";
 export const LAB_SCOPE = "https://request.msidlab.com/.default";
+export const LAB_KEY_VAULT_URL = "https://msidlabs.vault.azure.net";
+export const LAB_CERT_NAME = "LabAuth";
```

Comment:
Per @gladjohn: doesn't MSAL JS already install this cert to the build machine? @hectormmg

Reply:
If I remember correctly this was the case. there should be an install script in the YAML

Reply:
instead of writing to disk and using it, if the cert is already in the store, we can use it?

Reply:
Node has too many limitations in regards to cert functionality.